lib/application.js

```diff
@@ -23,6 +23,7 @@ class Application extends api.events.EventEmitter {
     this.config = new impress.Config(this);
     this.scheduler = new impress.Scheduler(this);
     this.schemas = new Map(); // Map of application schemas in metaschema format
+    this.latestSchema = null;
```

Comment:
Why do we have both `latestSchema` and `schemas` fields?

Reply:
So that we can easily determine the latest available schema.